airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/command/DestinationStream.kt

```diff
@@ -40,6 +46,60 @@ data class DestinationStream(
         fun toPrettyString() = "$namespace.$name"
     }
 
+    val schemaWithMeta: AirbyteType
```

Comment:
Should I just make all of the meta fields nullable?

Reply:
right now I guess we're only using it for tests and for writing new data, but it could break if db queries were built from this. (I was imagining we'd probably want a different approach for dbs tho?)

Reply:
yeah I think it's fine as-is, just wants some labelling as a "be careful" thing. I don't think dbs would use this to query existing data, but the fail case I can think of is a db destination trying to use this to decide the expected schema of a table (i.e. setting `not null` on some columns that should be nullable)